packages/components/breadcrumbs/src/breadcrumbs.ts

```diff
@@ -50,18 +50,14 @@ export class Breadcrumbs extends ScopedElementsMixin(LitElement) {
   /**
    * The url for the home link, defaults to the root url.
    *
-   * By changing this static property you can change the default value for
-   * all future instances of the component. Changing the static property
-   * won't affect already created instances.
+   * By changing this static property you can change the default value for all future instances of the component. Changing the static property won't affect already created instances.
```

Comment:
Why do this? It makes the comment less readable to have it all on 1 line?!

Reply:
Because it's rendering additional spaces in the markdown file.

Reply:
So maybe separated line should be added after the whole sentence and it would be better :)

Reply:
Hmm, can't you "collapse the whitespace" into a single space in the docs? I don't really like having a really long line in the code.

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/white-space-collapse ????

Reply:
it renders unnecessary `br`s in md file
<img width="708" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/9c06da87-cac2-4ef5-88d1-8789520a52d4">


But when I'll leave it like this:
<img width="1021" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/866f5d62-ca66-49fd-9596-08e9178ad222">

Looks quite ok in the comment and on the website:

<img width="960" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/114953cb-6f6d-4d9a-a3e2-3b6a48d21fa3">